dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java

```diff
@@ -356,6 +362,65 @@ void shouldOrderTrackedEntitiesByEnrolledAtAsc()
     assertEquals(List.of("QS6w44flWAf", "dUE514NMOlo"), trackedEntities);
   }
 
+  @Test
+  void shouldExcludeEnrollmentWhoseProgramIsNotRequestedAndOrderTrackedEntitiesByEnrolledAt()
+      throws ForbiddenException, BadRequestException, NotFoundException {
+
+    String oldEnrollment = "nxP7UnKhomJ";
```

Comment:
Can you please put the test setup into tracker/event_and_enrollment.json
This is now the only test that does it differently in this test class.